h264/read.go

```diff
@@ -0,0 +1,300 @@
+package h264
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+type BitReader struct {
```

Comment:
Comments?

Reply:
This isn't my code. This should have been a file rename - I must have done something weird.